src/model/index.ts

```diff
@@ -150,6 +149,16 @@ class Model implements WorkbookModel {
   ) {
     const calculator = new OutstandingStockPlanCalculator();
 
+    // get stock class for this plan
```

Comment:
The model is responsible to find the correct ratio for the preferred stock class related to a warrant or stock plan.
I previously put that responsibility in the columns, but it didn't make sense